### PR TITLE
[New Rule] Potential LSA Authentication Package Abuse

### LINKED ISSUE #859: [New Rule] Persistence via LSA Authentication Package
## Description
Adversaries can use the autostart mechanism provided by LSA authentication packages for persistence by placing a reference to a binary in the Windows Registry. The binary will then be executed by the system when the authentication packages are loaded.

## Required Info

- **Eventing Sources:**
`winlogbeat-*`, `logs-endpoint.events.*`

- **Platforms**
`windows`

## Optional Info
- **References:**

## Syntax
```
registry where event.type:"change" and
  registry.path: "HKLM\\SYSTEM\\*ControlSet*\\Control\\Lsa\\Authentication Packages"
```
Potential addition - `registry.data.type:"REG_BINARY"` but wouldn't be ECS compliant.

## Example Data

image

### PR DESCRIPTION
**Pull request by bm11100**
## Issues
resolves #859 

## Summary
Adversaries can use the autostart mechanism provided by the Local Security Authority (LSA) authentication packages for privilege escalation or persistence by placing a reference to a binary in the Windows registry. The binary will then be executed by SYSTEM when the authentication packages are loaded.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by threat-punter — COMMENTED** [review]
👍

File: rules/windows/privilege_escalation_lsa_auth_package.toml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+rule_id = "e9abe69b-1deb-4e19-ac4a-5d5ac00f72eb"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+registry where event.type : "change" and
+  registry.path : "HKLM\\SYSTEM\\*ControlSet*\\Control\\Lsa\\Authentication Packages" and
+  /* exclude SYSTEM SID - look for changes by non-SYSTEM user */
+  not user.id:"S-1-5-18"
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_lsa_auth_package.toml:28]:
```suggestion
  not user.id : "S-1-5-18"
```

Should this spacing be adjusted for consistency?

**Review by threat-punter — CHANGES_REQUESTED** [review]
I left a minor suggestion for your review. The rule needs to be linted too

File: rules/windows/privilege_escalation_lsa_auth_package.toml (hunk 1 above)
Comment (bm11100) [reply rules/windows/privilege_escalation_lsa_auth_package.toml:28]: yes it shouls

**Review by threat-punter — APPROVED** [review]

**Review by Samirbous — APPROVED** [review]

File: rules/windows/privilege_escalation_lsa_auth_package.toml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+rule_id = "e9abe69b-1deb-4e19-ac4a-5d5ac00f72eb"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+registry where event.type : "change" and
+  registry.path : "HKLM\\SYSTEM\\*ControlSet*\\Control\\Lsa\\Authentication Packages" and
+  /* exclude SYSTEM SID - look for changes by non-SYSTEM user */
+  not user.id : "S-1-5-18"
```
Comment (brokensound77) [inline rules/windows/privilege_escalation_lsa_auth_package.toml:28]:
```suggestion
registry where event.type == "change" and
  registry.path : "HKLM\\SYSTEM\\*ControlSet*\\Control\\Lsa\\Authentication Packages" and
  /* exclude SYSTEM SID - look for changes by non-SYSTEM user */
  not user.id : "S-1-5-18"
```

**Review by brokensound77 — APPROVED** [review]
minor tweak to query, then LGTM

**Pull request merged**